website/docs/getting-started/cluster-setup/kubernetes.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Kubernetes
+sidebar_position: 1
+---
+
+# Setting up a Kubernetes cluster
+*Note that this guide is only helpful in setting up a development environment; this setup is not recommended for a production environment.*
+
+## Prerequisites
+* You have a Kubernetes cluster setup, this could for example be a [minikube](https://minikube.sigs.k8s.io/docs/start/) cluster.
+* You have admin privileges to the cluster, since the Operator installation is only possible with an admin user.
+
+## Enabling Ingress
+To access an application externally, you will create _URLs_ using odo, which are implemented on a Kubernetes cluster by Ingress resources; installing an Ingress controller helps in using this feature on a Kubernetes cluster.
+
+**Minikube:** To install an Ingress controller on a minikube cluster, enable the **ingress** addon with the following command:
+```shell
+minikube addons enable ingress
+````
+To learn more about ingress addon, see [the documentation on Kubernetes website](https://kubernetes.io/docs/tasks/access-application-cluster/ingress-minikube/).
+
+*Other Kubernetes Cluster**: To enable the Ingress feature on a Kubernetes cluster _other than minikube_, using the NGINX Ingress controller see [the official NGINX Ingress controller installation documentation](https://kubernetes.github.io/ingress-nginx/deploy/).
```

Comment:
It is not clear what covers the **Other Kubernetes cluster**. It could be better to use a section:

```suggestion
### Other Kubernetes Cluster

To enable the Ingress feature on a Kubernetes cluster _other than minikube_, using the NGINX Ingress controller see [the official NGINX Ingress controller installation documentation](https://kubernetes.github.io/ingress-nginx/deploy/).
```

and idem for **minikube**



If you prefer to keep this presentation, please just add a missing \*